app/controllers/user/stores_controller.rb

```diff
@@ -2,4 +2,25 @@ class User::StoresController < ApplicationController
   def index
     @stores = current_user.stores
   end
+
+  def edit
+    @store = Store.find(params[:id])
+  end
+
+  def update
+    store = Store.find(params[:format])
```

Comment:
I've never seen format called in params. What is it?

Reply:
This format key actually held the store_id. I've never seen it before either but as I played around and inspected it, that's what it turned out to be - interestingly enough.